app/controllers/stories_bulk_update_controller.rb

```diff
@@ -0,0 +1,34 @@
+class StoriesBulkUpdateController < ApplicationController
+  def create
+    authorize stories
+
+    return render(json: { message: t(:stories_not_found) }, status: :not_found) if stories.blank?
+    @updater = StoryOperations::UpdateAll.call(stories, allowed_params, current_user)
+
+    returns_message @updater
+  end
+
+  private
+
+  def project
+    @project ||= Project.find(params[:project_id])
+  end
+
+  def stories
+    @stories ||= project.stories.where(id: params[:story_ids])
+  end
+
+  def allowed_params
+    params
+      .permit(:requested_by_id, :owned_by_id, :labels)
+      .merge(acting_user: current_user)
+  end
+
+  def returns_message(records)
+    if records
+      render json: { message: t(:update_stories_successfully) }, status: :ok
```

Comment:
We don't need that `status: :ok` here since it's the Rails' default.

Reply:
Okay, thanks!